bigframes/clients.py

```diff
@@ -162,3 +164,25 @@ def _get_service_account_if_connection_exists(
             pass
 
         return service_account
+
+
+def get_connection_name_full(
```

Comment:
[nit] since it takes a connection and returns a connection after resolving the format, would prefer naming it `resolve_full_connection_name`

Reply:
Will address in another PR.